dcc-storage-client/src/test/java/org/icgc/dcc/storage/client/ClientMainTest.java

```diff
@@ -86,12 +86,17 @@ public void testMainDownloadWithNonExistentManifest() throws Exception {
   @Test
   public void testMainDownloadWithNonExistentOutDir() throws Exception {
     val file = tmp.newFile();
-    val outDir = new File("/foo");
+    val outDir = new File("/tmp/foo");
```

Comment:
This test is to explicitly verify the directory creation logic, so I'm exercising finer control over directory specification process.